Refresh.GameServer/Endpoints/Game/UserEndpoints.cs

```diff
@@ -104,6 +97,12 @@ public SerializedUserList GetMultipleUsers(RequestContext context, GameDatabaseC
             return null;
         }
 
+        if (data.Description is { Length: > 4096 })
+        {
+            database.AddErrorNotification("Profile update failed", "Your profile failed to update because the description was too long. (Max 4096)", user);
+            return null;
+        }
```

Comment:
```suggestion
        const int maxDescriptionLength = 4096;
        if (data.Description is { Length: > maxDescriptionLength })
        {
            database.AddErrorNotification("Profile update failed", $"Your profile failed to update because the description was too long. The max length is {maxDescriptionLength}.", user);
            return null;
        }
```